lib/ApplicationFeatures/LanguageFeature.cpp

```diff
@@ -198,8 +198,10 @@ void LanguageFeature::prepare() {
   auto context = ArangoGlobalContext::CONTEXT;
   std::string binaryExecutionPath = context->getBinaryPath();
   std::string binaryName = context->binaryName();
-  _icuData = LanguageFeature::prepareIcu(_binaryPath, binaryExecutionPath, p,
-                                         binaryName);
+  if (_icuData.empty()) {
```

Comment:
Do we still need this change after fix for tests?

Reply:
Yes. If we launch 100 tests in a row it leads to high memory consumption

Reply:
But maybe just trigger cleanup of loaded IcuDataPtr inside Utf8Helper in test tear down ?